gpu-operator/dra-driver.rst

```diff
@@ -0,0 +1,492 @@
+
+Once scheduled to a node, the DRA driver adds a Node label for the ComputeDomain to the node where the workload has been scheduled to indicate the node is part of that ComputeDomain.
+This label is used as a NodeSelector on the DaemonSet mentioned above to trigger the scheduling of its pods to specific nodes.
+Once all workloads running in a ComputeDomain have run to completion, the label gets removed even if the ComputeDomain itself hasn't been deleted yet.
+This allows these nodes to be reused for other ComputeDomains and blocks the workloads from running until the IMEX daemon on that node has been started.
+
+The addition of this Node label triggers the DaemonSet to deploy an IMEX daemon to that node and start running it.
+
+
+Once all daemons have been fully started, the DRA driver unblocks each worker, injects its IMEX channel into the worker and allows it to start running.
+
+
+View the Compute Domain resources on your cluster
+=================================================
+
+
+#. View a ComputeDomain resource.
+
+   .. code-block:: console
+
+      $ kubectl describe computedomain/imex-channel-injection
+
+   *Example Output*
+
+   .. code-block:: output
+
+      Name:         imex-channel-injection
+      Namespace:    default
+      Labels:       <none>
+      Annotations:  <none>
+      API Version:  resource.nvidia.com/v1beta1
+      Kind:         ComputeDomain
+      Metadata:
+        Creation Timestamp:  2025-03-06T11:24:47Z
+        Finalizers:
+          resource.nvidia.com/computeDomain
+        Generation:        1
+        Resource Version:  18161752
+        UID:               e771ff54-948f-405d-8ef2-47ae83857500
+      Spec:
+        Channel:
+          Resource Claim Template:
+            Name:   imex-channel-0
+        Num Nodes:  1
+      Events:       <none>
+
+
+#. View IMEX channel creation logs.
+
+   .. code-block:: console
+
+      $ kubectl logs -n nvidia-dra-driver-gpu -l resource.nvidia.com/computeDomain --tail=-1
+
+   *Example Output*
+
+    .. literalinclude:: ./manifests/output/imex-logs.txt
+        :language: yaml
+  
+Node and Pod Affitnity Strategies
+=================================
+
+A ComputeDomain isn't strictly about IMEX channels—it's about running workloads across a group of compute nodes. 
+This means even if some nodes are not IMEX capable, they can still be part of the same ComputeDomain.
+To control where your workloads run, you should use NodeAffinity and PodAffinity rules.
+
+For example you could set PodAffinity with a preferred topologyKey set to ``nvidia.com/gpu.clique`` for workloads to span multiple NVLink domains but want them packed as tightly as possible. Or use a required topologyKey set to ``nvidia.con/gpu.clique`` when you require all workloads deployed into the same NVLink domain, but don't care which one. 
+
+
+******************************************************************
+Run a Multi-node nvbandwidth Test Requiring IMEX Channels with MPI

```

Comment:
i need to do some more testing to clean up this example and add some explanations around whats happening.